server/application-server/src/main/java/de/tum/in/www1/hephaestus/codereview/CodeReviewService.java

```diff
@@ -0,0 +1,212 @@
+package de.tum.in.www1.hephaestus.codereview;
+
+import java.io.IOException;
+import java.time.Instant;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+import org.kohsuke.github.GHPullRequest;
+import org.kohsuke.github.GHRepository;
+import org.kohsuke.github.GHUser;
+import org.kohsuke.github.GitHub;
+import org.kohsuke.github.GitHubBuilder;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Service;
+import org.springframework.web.client.RestClient;
+
+import org.springframework.data.domain.Example;
+import org.springframework.graphql.client.HttpSyncGraphQlClient;
+
+import de.tum.in.www1.hephaestus.EnvConfig;
+import de.tum.in.www1.hephaestus.codereview.actor.Actor;
+import de.tum.in.www1.hephaestus.codereview.actor.ActorConverter;
+import de.tum.in.www1.hephaestus.codereview.actor.ActorRepository;
+import de.tum.in.www1.hephaestus.codereview.comment.Comment;
+import de.tum.in.www1.hephaestus.codereview.comment.CommentConverter;
+import de.tum.in.www1.hephaestus.codereview.comment.CommentRepository;
+import de.tum.in.www1.hephaestus.codereview.pullrequest.Pullrequest;
+import de.tum.in.www1.hephaestus.codereview.pullrequest.PullrequestConverter;
+import de.tum.in.www1.hephaestus.codereview.pullrequest.PullrequestRepository;
+import de.tum.in.www1.hephaestus.codereview.repository.Repository;
+import de.tum.in.www1.hephaestus.codereview.repository.RepositoryConverter;
+import de.tum.in.www1.hephaestus.codereview.repository.RepositoryRepository;
+
+@Service
+public class CodeReviewService {
```

Comment:
Call it `GitHubDataSyncService` and put it together with the schedular in a directory